packages/lesswrong/server/recommendations/BestOfStrategy.ts

```diff
@@ -0,0 +1,29 @@
+import FeatureStrategy from "./FeatureStrategy";
+import type { StrategySpecification } from "../../lib/collections/users/recommendationSettings";
+
+/**
+ * A recommendation strategy that returns the highest voted posts that the user
+ * hasn't viewed. Note that, for performance reasons, the scores are not inflation
+ * adjusted.
```

Comment:
Potentially. It might not be worth it for this, but we could probably use that same view in frontpage queries. That may or may not give some performance benefit.